tests/python/unittest/test_meta_schedule_schedule_rule_multi_level_tiling.py

```diff
@@ -563,6 +563,53 @@ def test_multi_level_tiling_dense_dpa4():
     check_trace(spaces, expected)
 
 
+def test_multi_level_tiling_dense_dp4a_non_tensorizable():
+    m, n, k = 128, 128, 128
```

Comment:
would be great to also add a test case of non-divisible shape (m=n=k=127, dtype=int8)